src/main/java/taxi/service/ManufacturerServiceImpl.java

```diff
@@ -20,7 +20,7 @@ public Manufacturer create(Manufacturer manufacturer) {
     @Override
     public Manufacturer get(Long id) {
         return manufacturerDao.get(id).orElseThrow(() ->
-            new NoSuchElementException("Can't get manufacturer by id: " + id)
+                new NoSuchElementException("Can't get manufacturer by id: " + id)
```

Comment:
The line is indented with an extra 4 spaces. It should have the same indentation as the line above.